.github/workflows/cd.yml

```diff
@@ -71,15 +71,10 @@ on:
 
       # Build options.
       plugin-directory:
-        description: Directory of the plugin, if not in the root of the repository. If provided, package-manager must also be provided.
+        description: Directory of the plugin, if not in the root of the repository.
         type: string
         required: false
         default: .
-      package-manager:
-        description: The package manager to use.
-        type: string
-        required: false
-        default: ""
```

Comment:
This is a breaking change. Please rename the PR to:

```
feat(ci)!: use pm-detect to understand nodejs package manager
```

So release-please picks it up as a breaking change and releases a new major version